hs_core/tasks.py

```diff
@@ -260,4 +260,4 @@ def check_irods_files():
     This logs DEBUG messages for every problem it finds

```

Comment:
It is for future inclusion in celery chron. It is unused until we want automatic overnight checks.